labgrid/driver/dockerdriver.py

```diff
@@ -0,0 +1,86 @@
+import attr
+import docker
+import logging
+
+from labgrid.factory import target_factory
+from labgrid.driver.common import Driver
+from labgrid.resource.docker import DockerDaemon, DockerConstants
+
+
+@target_factory.reg_driver
+@attr.s(cmp=False)
+class DockerDriver(Driver):
```

Comment:
This Driver does not implement any of the Protocols.
It should implement at least one, I guess the PowerProtocol would be a good fit. This would allow one of the example strategies to start and stop the container.
If none of the available protocols fit this use case, please provide a valid use case and outline an example protocol.